### PR TITLE
DOCS-2070: Publishes OSS 3.27.2 documentation

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by ctauchen** [comment]
@danudey I've prepared the publication branch and publishing PR for our 3.27.1 release. 

Outstanding jobs:
* updates to releases.json (operator version for sure, maybe more)
* updates to variables.js (none? I think that's solid)
* updates to release notes (I've created the stub at the bottom of the page, but needs DATE and generated RN material).

File: calico_versioned_docs/version-3.27/release-notes/index.mdx [hunk 1]
```diff
@@ -117,17 +117,34 @@ December 18, 2023
 
 Calico Open Source release 3.27.0 is now generally available.
 
-<!--
-### Calico Open Source 3.18.1 bux fix update
+### Calico Open Source 3.27.2 bux fix release
```
Comment (sujeet-kr) [inline calico_versioned_docs/version-3.27/release-notes/index.mdx]:
```suggestion
### Calico Open Source 3.27.2 bug fix release
```

File: calico_versioned_docs/version-3.27/release-notes/index.mdx [hunk 2]
```diff
@@ -117,17 +117,34 @@ December 18, 2023
...
+
+February 16, 2024
+
+#### Bug fixes
```
Comment (sujeet-kr) [inline calico_versioned_docs/version-3.27/release-notes/index.mdx]: Should `Enhancements` be before `Bug fixes`?

File: calico_versioned_docs/version-3.27/release-notes/index.mdx (hunk 1 above)
Comment (danudey) [reply calico_versioned_docs/version-3.27/release-notes/index.mdx]: Fixed

File: calico_versioned_docs/version-3.27/release-notes/index.mdx [hunk 3]
```diff
@@ -117,17 +117,34 @@ December 18, 2023
 
 Calico Open Source release 3.27.0 is now generally available.
 
-<!--
-### Calico Open Source 3.18.1 bux fix update
+### Calico Open Source 3.27.2 bug fix release
 
-DD M YYYY
+February 16, 2024
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.27/release-notes/index.mdx]:
```suggestion
February 17, 2024
```
But we can fix this later

**Comment by ctauchen** [comment]
One small nit, otherwise LGTM, pending a clean build. Thanks @danudey !!

**Review by sujeet-kr — APPROVED** [review]

**Pull request merged**